code/modules/power/singularity/generator.dm

```diff
@@ -13,7 +13,8 @@
 /obj/machinery/the_singularitygen/process()
 	var/turf/T = get_turf(src)
 	if(src.energy >= 200)
-		new creation_type(T, 50)
+		var/obj/singularity/S = new creation_type(T, 50)

```

Comment:
Won't really break anything since you don't use any singuloth vars, fingerprint is either atom or object anyways.